lms/static/js/instructor_dashboard/util.js

```diff
@@ -401,6 +401,7 @@
         keywordValidator.keyword_regex = /%%+[^%]+%%/g;
 
         keywordValidator.keywords = [
+            '%%USERNAME%%', '%%COURSE_ID%%', '%%COURSE_START_DATE%%',
```

Comment:
So we forked this, right?
They don't support these keywords upstream?

Reply:
Correct